### PR TITLE
Update flux version to v0.50.2

### PR DESCRIPTION
**Pull request by jsternberg**
This upgrades the flux version to v0.50.2.

The secret service, which is used for alerts, is not included. The
`to()` function is also still not included.

### CODE REVIEW

File: storage/reads/reader_test.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+				return &cursors.IntegerArray{
+					Timestamps: []int64{10},
+					Values:     []int64{1},
+				}
+			}
+			cur.CloseFunc = func() {
+				closed++
+			}
+			return cur
+		}()
```
Comment (affo) [inline storage/reads/reader_test.go]: Do those 2 tables have the same group key?

That's why you expect to get only the first one?

Comment (jsternberg) [reply storage/reads/reader_test.go]: Ah yes. That must be what this is. I wasn't sure exactly what was going on here.

**Comment by dgnorton** [comment]
Looks like Circle hit a panic...
```
--- FAIL: TestDuplicateKeys_ReadFilter (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x856fd17]

goroutine 16 [running]:
testing.tRunner.func1(0xb64c6e0)
	/usr/local/go/src/testing/testing.go:830 +0x30e
panic(0x89960c0, 0x93efc40)
	/usr/local/go/src/runtime/panic.go:522 +0x16e
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/memory.Set(...)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/memory/memory.go:25
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array.(*builder).init(0xb438db0, 0x20)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array/builder.go:101 +0xc7
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array.(*Int64Builder).init(0xb438db0, 0x20)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array/numericbuilder.gen.go:102 +0x2f
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array.(*Int64Builder).Resize(0xb438db0, 0x2)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array/numericbuilder.gen.go:125 +0x42
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array.(*builder).reserve(0xb438db0, 0x1, 0xb451c64)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array/builder.go:138 +0x72
github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array.(*Int64Builder).Reserve(0xb438db0, 0x1)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/apache/arrow/go/arrow/array/numericbuilder.gen.go:113 +0x51
github.com/influxdata/influxdb/vendor/github.com/influxdata/flux/arrow.NewInt(0xb4410d0, 0x1, 0x1, 0x0, 0x89e9ca0)
	/root/go/src/github.com/influxdata/influxdb/vendor/github.com/influxdata/flux/arrow/int.go:10 +0x6c
github.com/influxdata/influxdb/storage/reads.(*floatTable).advance(0xb43a070, 0x0)
	/root/go/src/github.com/influxdata/influxdb/storage/reads/table.gen.go:91 +0x7e
github.com/influxdata/influxdb/storage/reads.newFloatTable(0xb444900, 0xe51600d8, 0xb40ea90, 0x0, 0x0, 0x1e, 0x0, 0x8c0acc0, 0xb438cc0, 0xb438cf0, ...)
	/root/go/src/github.com/influxdata/influxdb/storage/reads/table.gen.go:47 +0x1c2
github.com/influxdata/influxdb/storage/reads.(*filterIterator).handleRead(0xb478200, 0xb4106f8, 0x8c04060, 0xb477300, 0x0, 0x0)
	/root/go/src/github.com/influxdata/influxdb/storage/reads/reader.go:177 +0x755
github.com/influxdata/influxdb/storage/reads.(*filterIterator).Do(0xb478200, 0xb4106c8, 0xb49c010, 0x0)
	/root/go/src/github.com/influxdata/influxdb/storage/reads/reader.go:140 +0x138
github.com/influxdata/influxdb/storage/reads_test.TestDuplicateKeys_ReadFilter(0xb64c6e0)
	/root/go/src/github.com/influxdata/influxdb/storage/reads/reader_test.go:89 +0x1df
testing.tRunner(0xb64c6e0, 0x8acc61c)
	/usr/local/go/src/testing/testing.go:865 +0x97
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:916 +0x2b2
FAIL	github.com/influxdata/influxdb/storage/reads	0.045s
?   	github.com/influxdata/influxdb/storage/reads/datatypes	[no test files]
```

**Comment by jsternberg** [comment]
@affo the `to()` transformation and secret service are not things I plan to include. InfluxDB 1.x doesn't have an equivalent for the secret service, but we could implement the `to()` transformation and probably should. I think there's some work to have `to()` use the public write API and we should be able to take some of that work, adapt it to use the 1.x write API, and get `to()` working properly in 1.x.

**Pull request merged**